README.md

```diff
@@ -1,3 +1,9 @@
 # hw01
 
-My name is Mikayla Williams!
\ No newline at end of file
+# My name is Mikayla Williams and this is a bit about me. These are some things you should know about me:
+* I am a **second year** in the college, looking to major in Economics and minor in Urban studies.
+* I also enjoy:
+  + baking 
+  + playing rugby
+  + drinking coffee
+* Here is a picture of a dog, which I like more than cats: ![](https://78.media.tumblr.com/cb560d7310b5d997ba69433a1cb333fa/tumblr_o5u3ugVpda1vrwdf9o1_500.png)
```

Comment:
I know that you used a link for your image, but I would have liked to see how you could incorporate a link in another area of your text and practice using the text that you would click on for your hyperlink. I'm learning from you about the "Preview" option in Markdown, which will just be really helpful for me. Thank you for mentioning! I also had an early submit so I completely understand. Oops! 
In terms of this, I would have loved to learn more about you. You used all of the formats we were supposed to use, but maybe the heading could have been smaller (not an entire sentence) or the second sentence didn't need to be a part of the header. This might have helped give more structure to your overall bio.